test/test_serialization.py

```diff
@@ -493,6 +493,15 @@ def test_serialization_map_location(self):
         def map_location(storage, loc):
             return storage
 
+        def generate_map_locations(device_type):
+            return [
+                {'cuda:0': device_type + ':0'},
```

Comment:
```suggestion
                {device_type + ':0': device_type + ':0'},
```

Reply:
`cuda:0` on a left side of `{'cuda:0' : device_type + ':0'}` isn't a mistake. The example tensor we read on line 491 above is cuda generated, that's exactly where map_location comes handy - it allows to load tensor to other device memory bypassing original device which might not exist on the moment of execution. As far as I understand `{'cuda:0' : device_type + ':0'}` is a full signature of requested operation - we want tensor which is originally create for cuda to be loaded to other backend.

Reply:
you are right~